pkg/tidydns/tidydns.go

```diff
@@ -84,15 +112,248 @@ const (
 	RecordTypeSSHFP RecordType = 8
 	RecordTypeTLSA  RecordType = 9
 	RecordTypeCAA   RecordType = 10
+
+	AuthGroupUser       AuthGroup = 2
+	AuthGroupSuperAdmin AuthGroup = 1
 )
 
+const errorTidyDNS = "error from tidyDNS server: %s"
+const headerContentType = "Content-Type"
+const mimeForm = "application/x-www-form-urlencoded"
+
 type tidyDNSClient struct {
 	client   *http.Client
 	username string
 	password string
 	baseURL  string
 }
 
+func (c *tidyDNSClient) CreateInternalUser(ctx context.Context, username string, password string, description string, changePasswordOnFirstLogin bool, authGroup AuthGroup, userAllow []UserAllowID) (UserID, error) {
+	var userAllowFormatted []string
+	if len(userAllow) > 0 {
+		userAllowFormatted = make([]string, 0, len(userAllowFormatted))
+		for _, id := range userAllow {
+			userAllowFormatted = append(userAllowFormatted, strconv.Itoa(int(id)))
+		}
+	} else {
+		userAllowFormatted = []string{""}
+	}
+
+	var changePasswordOnFirstLoginFormatted string
+	if changePasswordOnFirstLogin {
+		changePasswordOnFirstLoginFormatted = "1"
+	} else {
+		changePasswordOnFirstLoginFormatted = "0"
+	}
```

Comment:
same as above:
```go
changePasswordOnFirstLoginFormatted := "0"
if changePasswordOnFirstLogin {
	changePasswordOnFirstLoginFormatted = "1"
}
```

Reply:
I would personally prefer the current version, but I don't have any strong feeling on changing it